docs/user/05-60-kyma-bindings.md

```diff
@@ -1,47 +1,86 @@
+> [!NOTE]
+>  Expired bindings do not count towards the bindings limit. However, they prevent creating new bindings until they exist in the database. Only after they are removed by the cleanup job or manually can the binding be recreated again.
+
+After the insert has been done, KEB creates a ServiceAccount, ClusterRole (admin privileges), and ClusterRoleBinding, all named `kyma-binding-{{binding_id}}`. The ClusterRole can be used to modify permissions granted to the kubeconfig.
+
+The created resources are then used to generate a [`TokenRequest`](https://kubernetes.io/docs/reference/kubernetes-api/authentication-resources/token-request-v1/). The token is then wrapped in a kubeconfig template and returned to the user. The encrypted credentials are then stored as an attribute in the previously created database binding.
+
+> [!NOTE]
+>  Creation of multiple and unused `TokenRequest` resources is not recommended
+
+
+## The Process of Fetching Kyma Binding Process
+
+![Get Binding Flow](../assets/bindings-get-flow.drawio.svg)
 
-To fetch a binding, use a GET request to the Broker API:
+The diagram shows a flow of fetching a Kyma binding in KEB. The process starts with a GET request sent to the KEB API. 
+The first instruction in the process is to check if a Kyma instance exists. If a Kyma instance exists, it must not be deprovisioned or suspended. 
+The endpoint doesn't return bindings for such instances. Existing bindings are retrieved by instance ID and binding ID. If any bindings exist, they are filtered by expiration date. KEB returns only non-expired bindings.
 
-```
-GET http://localhost:8080/oauth/v2/service_instances/{{instance_id}}/service_bindings/{{binding_id}}
-X-Broker-API-Version: 2.14
-```
+## The Process of Deleting a Kyma Binding
 
-The Broker returns the `200 OK` status code with the kubeconfig in the response body. If the binding does not exist, the instance does not exist, or the instance is suspended, the Broker returns a `404 Not Found` status code.
+![Delete Binding Flow](../assets/bindings-delete-flow.drawio.svg)
 
-## Unbinding
+The diagram shows the flow of removing a Kyma binding. The process starts with a DELETE request sent to the KEB API. The first instruction is to check if the Kyma instance that the request refers to exists. 
+Any bindings of non-existing instances are treated as orphaned and are destined to be removed. The next step is to conditionally delete the binding's `ClusterRole`, `ClusterRoleBinding`, and `ServiceAccount`, given that the cluster has been provisioned and not marked for removal. In case of deprovisioning or suspension of the Kyma cluster, this is unnecessary because the cluster is removed either way. 
```

Comment:
```suggestion
Any bindings of non-existing instances are treated as orphaned and are removed. The next step is to conditionally delete the binding's ClusterRole, ClusterRoleBinding, and ServiceAccount, given that the cluster has been provisioned and not marked for removal. In case of deprovisioning or suspension of the Kyma cluster, this is unnecessary because the cluster is removed either way. 
```